spec/orders.json

```diff
@@ -1,33 +1,45 @@
 {
-  "host": "api.shopbonsai.ca/orders",
-  "schemes": ["https"],
-  "basePath": "/",
+  "host": "api.shopbonsai.ca",
```

Comment:
Splitting host & basePath leads to fewer OpenAPI errors. This is autogenerated from the monorepo so we're good in the future